application/src/main/java/org/thingsboard/server/service/install/DefaultSystemDataLoaderService.java

```diff
@@ -298,6 +262,33 @@ public void createRandomJwtSettings() throws Exception {
         jwtSettingsService.createRandomJwtSettings();
     }
 
+    @Override
+    public void updateJwtSettings() {
+        JwtSettings jwtSettings = jwtSettingsService.getJwtSettings();
+
+        boolean invalidSignKey = false;
+
+        if (TOKEN_SIGNING_KEY_DEFAULT.equals(jwtSettings.getTokenSigningKey())) {
+            log.warn("WARNING: The platform is configured to use default JWT Signing Key. " +
+                    "Added new temporary JWT Signing Key. " +
+                    "This is a security issue that needs to be resolved. Please change the JWT Signing Key using the Web UI. " +
+                    "Navigate to \"System settings -> Security settings\" while logged in as a System Administrator.");
+            invalidSignKey = true;
+        } else if (Base64.getDecoder().decode(jwtSettings.getTokenSigningKey()).length * Byte.SIZE < 512) {
+            log.warn("WARNING: The platform is configured to use JWT Signing Key with length less then 512 bits of data. " +
+                    "Added new temporary JWT Signing Key. " +
+                    "This is a security issue that needs to be resolved. Please change the JWT Signing Key using the Web UI. " +
+                    "Navigate to \"System settings -> Security settings\" while logged in as a System Administrator.");
+            invalidSignKey = true;
+        }
+
+        if (invalidSignKey) {
+            jwtSettings.setTokenSigningKey(Base64.getEncoder().encodeToString(
+                    RandomStringUtils.randomAlphanumeric(64).getBytes(StandardCharsets.UTF_8)));
```

Comment:
```    
public void createRandomJwtSettings() {
        if (getJwtSettingsFromDb() == null) {
            log.info("Creating JWT admin settings...");
            this.jwtSettings = getJwtSettingsFromYml();
            if (isSigningKeyDefault(jwtSettings) || !validateTokenSigningKeyLength(jwtSettings)) {
                this.jwtSettings.setTokenSigningKey(Base64.getEncoder().encodeToString(
                        RandomStringUtils.randomAlphanumeric(64).getBytes(StandardCharsets.UTF_8)));
            }
            saveJwtSettings(jwtSettings);
        } else {
            log.info("Skip creating JWT admin settings because they already exist.");
        }
    }
```

we cant update settings from DB using this method